### PR TITLE
Made PlanManagementTest create actual models instead of using phantom IDs

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/shoebox/marketing [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 1b60c1b05ea05ca1775d01df77921db9affde836
+Subproject commit 6e323987276112be146e8302aa9b78dd91c93506
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/marketing:1]: this?

**Comment by leogrim** [comment]
Good catch thanks!

File: kifi-backend/modules/shoebox/marketing (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/marketing:1]: I think that someone had a bad merge. Daily reminder that `git rebase` doesn't have this issue :neutral_face:

**Comment by ryanpbrewster** [comment]
I'm like 75% sure this was the issue (because there is a referential integrity constraint on `paid_account.org_id` and the org didn't actually exist.

What actually terrifies me is that it ever passed. This was not a heisentest. It was either okay or straight up broken.

**Pull request merged**